docs/user-guide/plugins/copacetic.md

```diff
@@ -0,0 +1,56 @@
+# Copacetic
+
+## Introduction
+The Copacetic plugin of Devtron helps you patch your container image vulnerabilities traced by the security scan Devtron performed on your container image. By integrating the **Copacetic** plugin into your workflow and enabling the **Scan for vulnerabilities** at your **Build stage**, you can:
+- Trace the vulnerabilities of your container images, and the **Copacetic** plugin will automatically patch the container image vulnerabilities for you.
+
+### Prerequisites
+Before integrating the **Copacetic** plugin, install the `Vulnerability Scanning (Trivy/Clair)` integration from Devtron Stack Manager. Once the integration is installed, make sure you have enabled **Scan for vulnerabilities** at the **Build stage** or integrated the **Code-Scan** plugin in the **Pre-build stage**.
+
```

Comment:
@bhushan-nemade-dt, In this line, we seem to have put 'Code-Scan' in bold: `integrated the **Code-Scan** plugin`
Can we hyperlink it to our existing `Code-Scan` doc, instead?

Reply:
sure, linked the code-scan plugin at prerequisites